ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -0,0 +1,187 @@
+import {Component} from '@angular/core';
```

Comment:
I broke the change history a bit by naming the new admin page `admin-user` instead of `admin-single-user` or something like that. I can change this if you think it's a problem.

Reply:
Added an anchor on the user's name to go to the single-user page.